### PR TITLE
Fix for iCloud path change in Catalina

### PR DESCRIPTION
**Pull request by rsoper**
Using platform to check for macOS version and setting icloud_home to the correct path accordingly. 
Revolves #1386

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,6 @@
 
 # Used by pyenv
 /.python-version
+
```
Comment (pvinis) [inline .gitignore]: these lines can be in another pr maybe.

**Comment by rsoper** [comment]
Need to retract this temporarily.
